auth0/src/main/java/com/auth0/android/authentication/storage/SecureCredentialsManager.kt

```diff
@@ -340,6 +365,9 @@ public class SecureCredentialsManager @VisibleForTesting(otherwise = VisibleForT
         if (scope != null) {
             request.addParameter("scope", scope)
         }
+
+        request.addParameters(parameters)
```

Comment:
might make sense to move this line above the "add scope" line, so that particular parameter always has precedence

Reply:
done